services/github-service-api/src/main/java/org/kontinuity/catapult/service/github/api/GitHubService.java

```diff
@@ -54,4 +54,13 @@ GitHubWebhook createWebhook(GitHubRepository repository,
                                 GitHubWebhookEvent... events)
             throws IllegalArgumentException;
     
+    /**
+     * Deletes all webhooks in a specific GitHub repository
+     * 
+     * @param repository - the value object that represents the GitHub repository
+     * @throws IOException
+     * @throws IllegalArgumentException If the parameter is unspecified
+     */
+    void deleteWebhooks(final GitHubRepository repository) throws IOException, IllegalArgumentException;
```

Comment:
1) Has to go into an SPI interface; we don't expose this to users/callers of the API...it's just for our use in testing
2) Why delete all webhooks?  Shouldn't we be deleting only the one we created?
3) Let's wrap any `IOException` as it occurs in an appropriate `RuntimeException`; the caller can't be expected to handle a low-level event like that

`boolean deleteWebhook(final GitHubWebhook webhook) throws IllegalArgumentException`